Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'safemode', '~> 1.2'
 gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
+gem 'rails-i18n', '~> 3.0.0'
```

Comment:
Wild idea - can we make '3.0.0' a constant like `RAILS_VERSION` so it's more obvious? Maybe we have more gems which are in sync with Rails version, couldn't find any tho.